addons/sourcemod/configs/szf/weapons.cfg

```diff
@@ -683,5 +683,10 @@
 			"text"		"Melee_CloakandDagger"
 			"weapon"	"30"	//Invis Watch
 		}
+		"173"	//Vita-Saw
+		{
+			"text"		"Melee_VitaSaw"
+			"attrib"	"10 ; 1.15 ; 811 ; 0.0"
+		}
```

Comment:
Move between Ubersaw and Amputator changes to keep order constant.
Same to translation